cmd/go-getter/main.go

```diff
@@ -61,8 +61,8 @@ func main() {
 	client := getter.DefaultClient
 
 	getters := getter.Getters
-	getters["gcs"] = new(gcs.Getter)
-	getters["s3"] = new(s3.Getter)
+	getters = append(getters, new(gcs.Getter))
+	getters = append(getters, new(s3.Getter))
```

Comment:
Do we want to add the smb getter as as default getter cmd here ? ( I would think so )

Reply:
They are already in the `getter.Getters` as the rest of the Getters

Reply:
Right !